opensearchpy/__init__.py

```diff
@@ -95,6 +97,7 @@
     "OpenSearchWarning",
     "OpenSearchDeprecationWarning",
     "AWSV4SignerAuth",
+    "AWSV4SignerAuthAsync",
```

Comment:
Should it be `AWSV4SignerAsyncAuth` to match naming in `RequestsAsyncHttpConnection`?

Reply:
Yep, I updated the naming on that. I updated the connection class to `AsyncHttpConnection` as well because I realized it was actually not using the requests library.